package.json

```diff
@@ -13,5 +13,8 @@
     "tests": "./scripts/start-containers-for-tests.sh",
     "analytics": "docker compose -f ./docker-compose.yml -f ./docker-compose.local.yml --profile mock-services --profile analytics  up -d --quiet-pull",
     "logs": "docker compose logs --tail 30 -f"
+  },
+  "dependencies": {
+    "@types/geojson": "^7946.0.14"
```

Comment:
This looks like an `npm i` at the wrong level

Reply:
Fixed by [c410ab5](https://github.com/theopensystemslab/planx-new/pull/3378/commits/c410ab5a1fd45e150c86baf527063a0dcb869e72)